distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/xml/WFS_11_sfRoads.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
```

Comment:
✏️ technically we shouldn't stick codice copyright on data that isn't our own (in this case we just grabbed it from geoserver).  We could add an exception so checkstyle skips these files.

Reply:
Done